app/scripts/utils/show-mouse-position.js

```diff
@@ -50,7 +52,7 @@ const showMousePosition = (
    * @param  {Boolean}   isNoClear  If `true` do not clear the graphics.
    */
   const drawMousePosition = (mousePos, isHorizontal, isNoClear) => {
-    if (!isNoClear) graphics.clear();
+    if (!isNoClear) clearMousePosition();
```

Comment:
To follow up on the comment above, why is it necessary to put the graphics.clear() call in a separate function. IMO it's a little misleading.

If you want to explain that the `graphics.clear()` call is used to clear the mouse position, I think a comment would be better suited.

Reply:
```suggestion
    if (!isNoClear) graphics.clear();
```

Leftover. Good catch.